packages/base-driver/lib/basedriver/driver.ts

```diff
@@ -309,6 +309,11 @@ export class BaseDriver<
 
     this._log.prefix = helpers.generateDriverLogPrefix(this);
 
+    this.log.updateCurrentContext({
+      sessionId: this.sessionId,
+      sessionSignature: this.sessionId.substring(0, 8),
```

Comment:
Please extract the method for session signature calculation into a common place, so we don't hardcode the same here and in middlewares

Reply:
I extracted to 'lib/helpers/session.ts' as calcSignature().